RoosterPlanner.UI/src/app/pages/admin/admin.component.ts

```diff
@@ -13,33 +16,53 @@ import{CreateProjectComponent} from "../../components/create-project/create-proj
 })
 export class AdminComponent implements OnInit {
   projects: Project[] = [];
-  listOfProjects:Project[][]=[]
-  tempList:Project[]=[];
+  listOfProjects: Project[][] = []
+  listOfAdmins: User[][] = []
```

Comment:
De 2D array is mij niet helemaal duidelijk. Wat gaat het doel hiervan zijn?

Reply:
Het idee is om een list van lists van type Project te hebben. 
Was waarschijnlijk te omslachtig.
het is nu aangepast naar:
  listOfProjects: any = []
  listOfAdmins: any = []